### ISSUE TITLE
Callable function don't work on emulator since 10.3.1

### ISSUE THREAD

**Issue by gpfister** [firebase/firebase-tools#4320]
### [REQUIRED] Environment info

**firebase-tools:** 10.3.1

**Platform:** Ubuntu 20.04

### [REQUIRED] Test case

Call a callable function (using the regular API, not the Admin API) using firebase-tools 10.3.1 (it works on 10.3.0).

### [REQUIRED] Steps to reproduce

Expose a callable function on the emulator, create a user and call the function after authenticating.

### [REQUIRED] Expected behavior

The function should be called.

### [REQUIRED] Actual behavior

The function doesn't get called, an unhandled execption.

```
⚠  functions: TypeError: Cannot read properties of undefined (reading 'data')
    at processBackground (./node_modules/firebase-tools/lib/emulator/functionsEmulatorRuntime.js:550:24)
    at invokeTrigger (./node_modules/firebase-tools/lib/emulator/functionsEmulatorRuntime.js:640:19)
    at handleMessage (./node_modules/firebase-tools/lib/emulator/functionsEmulatorRuntime.js:742:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
⚠  Your function was killed because it raised an unhandled error.
```

**Comment by sceee** [comment]
This was actually already adressed (and probably fixed) in #4314 .

**Comment by gpfister** [comment]
@sceee, thanks, I this is also a duplicate of #4313.

**Comment by taeold** [comment]
Fix is shipped in 10.4.0!
